lazy_tensor_core/lazy_tensor_core/csrc/ts_backend/aten_ltc_ts_type.cpp

```diff
@@ -72,6 +72,13 @@ at::Tensor subtensor(const at::Tensor& tensor, int dim, int groups, int g) {
 
 }  // namespace
 
+at::Tensor LazyNativeFunctions::_log_softmax(const at::Tensor& self, int64_t dim,
+                                         bool /* half_to_float */) {
```

Comment:
@asuhan I have followed the _softmax method to omit the half_to_float parameter, but not exactly sure why it can be omitted. Will appreciate it if you can illustrate a little bit.

Reply:
good question.  Half was not supported in LTC previously, but @desertfire just added it.  Maybe softmax needs to be updated too and both should support it.  

I guess for XLA it was not used, but it may be important for nvfuser which is obviosuly cuda-based.  

A quick search shows it is used basically for this special case of type-promotion in cuda, otherwise it is false:
https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/SoftMax.cpp#L29

Reply:
Filed #63113 to keep track of adding the support of the parameters since it might be better to do it for both operators and therefore is better as a follow up. Thanks for the illustration, @wconstab.